index.html

```diff
@@ -0,0 +1,118 @@
+        <span class="home-subtitle">"From Mendoza to the World!"</span>
+        <h1 class="home-title">La Rienda <br>World Tour 2023</h1>
+      </div>
+      <div class="home-description-container">
+        <p class="home-description">A fun festival where people can listen this great band and start knowing the Argentinian's native music. A complete show with dancers, lights and many FX that will surprise you</p>
+      </div>
+      <div class="home-place-container">
+        <p class="home-date">NEXT: 2022.11.11 (Fri)</p>
+        <p class="home-place">@ Movistar Arena, Bs. As.</p>
+      </div>
+    </section>
+    <section class="program-container">
+      <h2 class="programs-title">Tour Program</h2>
+      <hr class="title-separator">
+      <ul class="program-list">
+        <li class="program-card">
+          <img class="program-icon" src="./images/argentina.svg" alt="program icon">
+          <h3 class="program-title">Argentina</h3>
+          <p class="program-description">@ Aconcagua Arena<br>@ Movistar Arena</p>
+        </li>
+        <li class="program-card">
+          <img class="program-icon" src="./images/mexico.svg" alt="program icon">
+          <h3 class="program-title">Mexico</h3>
+          <p class="program-description">@ Arena CDMX<br>@ Arena Monterrey</p>
+        </li>
+        <li class="program-card">
+          <img class="program-icon" src="./images/united-states-of-america.svg" alt="program icon">
+          <h3 class="program-title">USA</h3>
+          <p class="program-description">@ UBS Arena<br>@ FTX Arena</p>
+        </li>
+        <li class="program-card">
+          <img class="program-icon" src="./images/spain.svg" alt="program icon">
+          <h3 class="program-title">Spain</h3>
+          <p class="program-description">@ WiZink Center<br>@ Navarra Arena</p>
+        </li>
+        <li class="program-card">
+          <img class="program-icon" src="./images/japan.svg" alt="program icon">
+          <h3 class="program-title">Japan</h3>
+          <p class="program-description">@ Saitama Super Arena<br>@ Nippon Budokan</p>
+        </li>
+      </ul>
+    </section>
+    <section class="speakers-container">
+      <h2 class="speakers-title">Staff</h2>
+      <hr class="title-separator">
+      <div class="speakers-content">
+      </div>
+    </section>
+    <section class="partners-container">
+      <h2 class="partners-title">Partners</h2>
+      <hr class="title-separator">
+      <ul class="partners-list">
+        <li class="partner-card"><a href="#"><img class="partner-image" src="./images/Spotify.png" alt="partner logo"></a></li>
+        <li class="partner-card"><a href="#"><img class="partner-image" src="./images/instagram-white.png" alt="partner logo"></a></li>
+        <li class="partner-card"><a href="#"><img class="partner-image" src="./images/amazon-music.png" alt="partner logo"></a></li>
+      </ul>
+    </section>
+  </main>
+  <footer class="footer-container">
```

Comment:
- Optional
The footer of your application should be personalized as per your project. Currently, the logo and description in your footer are from the original design, when it should be the logo of your application along with its details. It would make much more sense that way. Also, since you will be putting this project on your portfolio, it is super important to keep things relevant to your project.